File: problems/algorithms/2011/solution.cpp
class Solution
{
public:
	int finalValueAfterOperations(vector<string> &operations)
	{
		int val = 0;
		for (string operation : operations)
		{
			if (operation[1] == '+')
				val++;
			else
				val--;
		}
		return val;
	}
};

File: problems/algorithms/2011/solution.rs
impl Solution {
    pub fn final_value_after_operations(operations: Vec<String>) -> i32 {
        let mut val = 0;
        for operation in operations.iter() {
            if operation.chars().nth(1).unwrap() == '+' {
                val += 1;
            } else {
                val -= 1;
            }
        }
        val
    }
}